__tests__/e2e/message.spec.js

```diff
@@ -0,0 +1,91 @@
+const uuid = require('uuid');
+const { pgClient, pgClient2 } = require('../../server/infra/database/pgClient');
+
+// test to confrim successful clients' subscription to a postgres channel
+describe('subscribes clients to postgres channel "new_tree_notification" and adds new tree record to confirm subscription', () => {
+    afterAll((done) => { // eslint-disable-line
+        pgClient.end();
+        pgClient2.end();
+        done();
+    })
+
+    // creates two clients and confirms their subscription to a postgres channel
+    it('subscribes two cilents', async () => {
+
+        //  tree object to be added to table
+        const treeObj = {
+            latest_capture_id: uuid.v4(),
+            age: 1
+        }
+
+        // query to create notification function on postgres schema
+        let funQueryRes;
+        try {
+            funQueryRes = await pgClient.query(`CREATE OR REPLACE FUNCTION queue.new_tree_notify_trigger() RETURNS TRIGGER AS $$
```

Comment:
@dagmawig you are using `trigger` and I don't think this is good for our pub/sub model, it bind the message to the business model( trees here), for a pub/sub, it is business model agnostic and could be sent message by any client (node.js microservice), but trigger just work for tables and any db operation.

What I imagine is more like this topic: https://medium.com/@simon.white/postgres-publish-subscribe-with-nodejs-996a7e45f88

The rough test steps:
1. Create notify channel
2. To client to subscribe to the channel
3. Send message to channel by this test.
4. All the two clients received the message.